htdocs/src/AppBundle/Controller/PageController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\PageGroup;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class PageController
+ *
+ * @package AppBundle\Controller
+ * @author Nick Lubisch <oc@nicklubisch.de>
```

Comment:
I think we didn't have this tag in oc.de or?
Feel free to create the author section in htdocs/composer.json.

Reply:
Tell me what to do :stuck_out_tongue_closed_eyes: its just my common class file template